### PR TITLE
Get bug fixes from version1.2 into main

### PR DESCRIPTION
**Pull request by dharmez**
Merge commits from version1.2 branch into main.

### CODE REVIEW

File: components/processing/form_parser/deployment/main.tf [hunk 1]
```diff
@@ -12,6 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+locals {
+  dpu_label = {
```
Comment (eyalbenivri) [inline components/processing/form_parser/deployment/main.tf:16]: Do we want to change the variable name as well?

File: components/dpu-workflow/src/batch_pdf_classifier.py [hunk 2]
```diff
@@ -16,12 +16,14 @@
...
 from typing import Optional
 
+from google.api_core.gapic_v1.client_info import ClientInfo  # type: ignore
 from google.api_core.client_options import ClientOptions # type: ignore # pylint: disable = no-name-in-module # pylint: disable = import-error
 from google.api_core.exceptions import InternalServerError # type: ignore # pylint: disable = no-name-in-module # pylint: disable = import-error
 from google.api_core.exceptions import RetryError # type: ignore # pylint: disable = no-name-in-module # pylint: disable = import-error
 from google.cloud import documentai  # type: ignore # pylint: disable = no-name-in-module # pylint: disable = import-error
 from google.cloud import storage # type: ignore # pylint: disable = no-name-in-module # pylint: disable = import-error
 
+USER_AGENT = "cloud-solutions/eks-docai-v1"
```
Comment (eyalbenivri) [inline components/dpu-workflow/src/batch_pdf_classifier.py:26]: Side note - in v1.2, this `batch_pdf_classifier.py` is NOT being used, but this change is fine nonetheless.

**Review by eyalbenivri — COMMENTED** [review]
Overall looks good - Seems like there are conflicts
Let's also rebase later to `rc1.3` so we won't lose these changes.

File: components/dpu-workflow/src/batch_pdf_classifier.py (hunk 2 above)
Comment (dharmez) [reply components/dpu-workflow/src/batch_pdf_classifier.py:26]: Yes, your are right.

File: components/processing/form_parser/deployment/main.tf (hunk 1 above)
Comment (dharmez) [reply components/processing/form_parser/deployment/main.tf:16]: It depends on level of effort. We can look at making those refactoring in v1.3 or beyond.

**Review by eyalbenivri — APPROVED** [review]

**Review by anuradha-bajpai-google — APPROVED** [review]

**Pull request merged**